requirements.txt

```diff
@@ -1,35 +1,54 @@
-#
-# Requirements for wger for production
-#
-
-Django>=1.9,<1.10
-django-recaptcha
-reportlab
-django_mobile
-django-formtools>=1.0,<1.1
-bleach
-python-mimeparse
-pillow
-easy-thumbnails
+alabaster==0.7.10
+Babel==2.5.0
+bleach==2.0.0
+certifi==2017.7.27.1
+chardet==3.0.4
+defusedxml==0.5.0
+Django==1.9.13
+django-appconf==1.0.2
+django-bootstrap-breadcrumbs==0.8.2
+django-bower==5.2.0
+django-compressor==2.2
+django-cors-headers==2.1.0
+django-filter==1.0.4
+django-formtools==1.0
+django-mobile==0.7.0
+django-recaptcha==1.3.1
+django-sortedm2m==1.5.0
+django-tastypie==0.13.3
+djangorestframework==3.2.5
+docutils==0.14
+easy-thumbnails==2.4.1
 fitbit==0.3.0
-django_compressor
-icalendar
-django-bootstrap-breadcrumbs
-sphinx
-django-sortedm2m
-django-bower
-invoke
-requests
-whitenoise==3.3.0
 gunicorn==19.7.1
-
-# REST API
-djangorestframework>=3.2,<3.3
-django-filter
-django-tastypie
-django-cors-headers
-
-# Python3 compatibility
-six
-
+html5lib==0.999999999
+icalendar==3.11.6
+idna==2.6
+imagesize==0.7.1
+invoke==0.14.0
+Jinja2==2.9.6
+MarkupSafe==1.0
+oauthlib==2.0.2
+olefile==0.44
+Pillow==4.2.1
+Pygments==2.2.0
+PyJWT==1.5.3
+python-dateutil==2.6.1
+python-mimeparse==1.6.0
+python3-openid==3.1.0
+pytz==2017.2
```

Comment:
avoid using pip freeze, add the new dependency manually to requirements.txt

Reply:
Alright